### DISCUSSION TITLE
Can passwords to be sent to avoid Chia being stolen?

### DISCUSSION THREAD

**Discussion by abenlee** [Chia-Network/chia-blockchain#14891] [Support]
### What happened?

Can passwords to be sent to aviod chia to be stolen? Many person's chia has been stolen.

### Version

1.7

### What platform are you using?

Windows

### What ui mode are you using?

GUI

### Relevant log output

```shell
Can passwords to be sent to aviod chia to be stolen? Many person's chia has been stolen.
```

**Comment by ZeroCool1974-BR** [comment]
Send your XCH to Goby or Pawket, I personally leave only some mojos on the farmer wallet for safety!
Web 3.0, this is the way! ;-)

**Comment by MumfMeisterT** [comment]
This seems more like a discussion/support topic. Transferring issue to Discussion board. -- You may also reach out to us and the community by messaging us on Keybase in the chia_network.public team under the #Support channel.

**Comment by MumfMeisterT** [comment]
Do you know what software these other people are using on their machines where they are getting their XCH stolen?
